src/test/java/com/klarna/hiverunner/examples/rule/HelloAnnotatedHiveRunnerRule.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (C) 2013-2019 Klarna AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.klarna.hiverunner.examples.rule;
```

Comment:
I think this package should be name: `com.klarna.hiverunner.examples.junit4`

Reply:
We/I chose `rule` as it felt a bit more generic and didn't really want to mention a specific version of junit in the package name as you can still use rules in future versions of junit. On the other hand it is a junit4 feature and is tied to that version so I'm not too bothered if you think this makes it clearer.